apps/allocations/app/components/Panel/NewAllocation.js

```diff
@@ -1,16 +1,9 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
+import { BigNumber } from 'bignumber.js'
```

Comment:
Not a big deal ATM, but highly recommend using [BN.js](https://github.com/indutny/bn.js/) instead.

Reply:
Why exactly do you recommand BN.js, in my experience BigNumber has a richer feature set.

Reply:
That's true, there are specific cases where `BigNumber.js` is much easier to work with, but `BN.js` is faster and most implementations are moving to it.